app/src/main/res/layout/fragment_transaction_list.xml

```diff
@@ -45,6 +56,7 @@
                 android:textSize="30dp"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/textView2" />
+
```

Comment:
nit - extra whitespace, good to remove 

^ also if I say nit feel free to skip it if it's going to hold you up at all on git :)